Marlin/language_de.h

```diff
@@ -131,7 +131,7 @@
 #define MSG_RESUMING                        "Druckfortsetzung"
 #define MSG_PRINT_ABORTED                   "Druck abgebrochen"
 #define MSG_NO_MOVE                         "Motoren eingeschaltet"
-#define MSG_KILLED                          "KILLED"
+#define MSG_KILLED                          "ABGEBROCHEN"
 #define MSG_STOPPED                         "ANGEHALTEN"
 #define MSG_CONTROL_RETRACT                 "Retract mm"
```

Comment:
The word Retract is not german. Maybe we can use something with "zurückziehen" or "einziehen"